docs/cities/manchester/teaching-resources.md

```diff
@@ -0,0 +1,39 @@
+---
+id: teaching-resources
+title: Teaching Resources
+---
+
+> **Note:**
+>
+> Resoures here is quite new and recent. We are trying different ways to consolate our collective knowledge, it will be subjected to change.
+
+Teaching resources is used as a single source of truth that all members of CYF can access.
+
+## HTML & CSS
+
+### Week 1
+
+#### Lesson Information
+
+- [Lesson Week 1 [GitHub]](https://github.com/anthonytranDev/cyf-html-css/blob/master/week-1/LESSON.md)
+
+#### Presentation Slides
+
+- [Week 1 HTML & CSS [Slides]](https://docs.google.com/presentation/d/1StXwTWrTUXUknUbhdsKWaaGw4iemMxspahP1RUK6bgg/edit?usp=sharing)
+
+#### VS Code Basics
+
+- [Visual Studio Code Basics [Doc]](https://docs.google.com/document/d/13ptS_akzgCQuBGQXzvoBpdOazeg8L7eDIE_cBGGaWsI/edit?usp=sharing)
+
+#### HTML & CSS
+
+- [HTML & CSS - CYF [GitHub]](https://syllabus.codeyourfuture.io/html-css/week-1/lesson.html)
```

Comment:
This should be the very first thing in the content for Week 1 - everything else flows from this.